build/charts/theia/templates/_helpers.tpl

```diff
@@ -79,3 +79,17 @@
     sizeLimit: {{ $clickhouse.storage.size }}
 {{- end }}
 {{- end }}
+
+{{- define "clickHouseMontiorImageTag" -}}
```

Comment:
you have the same typo in a couple of places: s/Montior/Monitor

Reply:
Thanks! Fixed.